python/ql/test/library-tests/frameworks/jmespath/taint_test.py

```diff
@@ -0,0 +1,33 @@
+import jmespath
+
+def test_taint():
+    data = TAINTED_DICT
+
+    expression = jmespath.compile("foo.bar")
```

Comment:
Calling these `unsafe_data` and `safe_expression` respectively would strengthen the symmetry and make the point clear (that only the data's safety is important).

Reply:
I don't really agree that this is too much of a problem, but it was easy to fix, so here you are :blush:

Reply:
I would not say it was a problem either, rather an opportunity :-) Thanks for the update.